### PR TITLE
Fix flakey twin e2e tests

### PR DESCRIPTION
**Pull request by timtay-microsoft**
A lot of scenarios had a 3 second timeout which is rather strict. Opening that timeout up to 1 minute so that the test isn't as flakey

### CODE REVIEW

File: iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/setup/DeviceTwinCommon.java [hunk 1]
```diff
@@ -103,9 +103,9 @@ public static Collection inputs() throws Exception
 
     // Max time to wait to see it on Hub
     protected static final long PERIODIC_WAIT_TIME_FOR_VERIFICATION = 1000; // 1 sec
-    protected static final long MAX_WAIT_TIME_FOR_VERIFICATION = 3000; // 30 sec
```
Comment (timtay-microsoft) [inline iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/setup/DeviceTwinCommon.java:106]: This was labeled as 30 seconds, but it was actually only 3 seconds

Comment (azabbasi) [reply iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/setup/DeviceTwinCommon.java:106]: And I assume 3 seconds wasn't long enough and extending the wait period will fix the flakiness?

Comment (timtay-microsoft) [reply iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/setup/DeviceTwinCommon.java:106]: Yeah

**Pull request merged**